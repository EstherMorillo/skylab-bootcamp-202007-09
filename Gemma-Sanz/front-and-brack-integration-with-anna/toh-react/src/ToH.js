import React from 'react';
import './bootstrap.min.css';
import './ToH.css';
import Dashboard from './ components/dashboard';
import Details from './ components/details';
import List from './ components/list';
import heroList from './ components/list.mock';
import Header from './ components/header';
import { Route, Switch } from 'react-router-dom';
import PageNotFound from './ components/PageNotFound';

function ToH(props) {
	console.log(heroList);
	return (
		<>
			<Header />
			<Switch>
				<Route path="/" exact component={Dashboard} />
				<Route path="/hero/:heroId" component={Details} />
				<Route path="/hero" component={Details} />
				<Route path="/heroes" component={List} />
				<Route component={PageNotFound} />
			</Switch>
		</>
	);
}

export default ToH;
