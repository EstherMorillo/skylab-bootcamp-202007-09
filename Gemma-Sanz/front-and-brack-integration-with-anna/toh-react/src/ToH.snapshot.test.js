import renderer from 'react-test-renderer';
import React from 'react';
import ToH from './ToH';
import { BrowserRouter as Router } from 'react-router-dom';

describe('App snapshot', () => {
	const tree = renderer.create(
		<Router>
			<ToH />
		</Router>
	);

	it('should match', () => {
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
