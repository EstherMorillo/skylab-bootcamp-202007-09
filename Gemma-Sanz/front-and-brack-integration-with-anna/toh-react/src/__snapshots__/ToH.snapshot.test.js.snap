// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should match 1`] = `
Array [
  <header>
    <h1>
      Tour of Heroes
    </h1>
    <nav>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Dashboad
      </a>
      <a
        aria-current={null}
        href="/heroes"
        onClick={[Function]}
      >
        Heroes
      </a>
    </nav>
  </header>,
  <div>
    <h2>
      Top four
    </h2>
    <ul
      className="dashboard_s_list row"
    />
  </div>,
]
`;
